src/index.js

```diff
@@ -131,14 +134,7 @@ async function run (cwd, config) {
         extends: [
           getConfig('standard', 'index.js'),
           getConfig('standard-jsx', 'index.js'),
-          getConfig('prettier', 'index.js'),
-          getConfig('prettier', '@typescript-eslint.js'),
-          getConfig('prettier', 'babel.js'),
-          getConfig('prettier', 'flowtype.js'),
-          getConfig('prettier', 'react.js'),
-          getConfig('prettier', 'standard.js'),
-          getConfig('prettier', 'unicorn.js'),
-          getConfig('prettier', 'vue.js')
```

Comment:
This addresses [the breaking changes](https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21) from v8.0.0 of `eslint-config-prettier`